script.js

```diff
@@ -0,0 +1,75 @@
+// Select the container
```

Comment:
Consider separating the styles from JavaScript code

Reply:
Because our session topic was DOM and styling in JS I think maybe it is better to style in JS. So do you recommend me to move the style to the CSS file?

Reply:
It's always good to have styles separated. unless it is a change that happens in the functions

Reply:
Thanks Reza, I will!